src/libraries/Microsoft.Extensions.Configuration.Abstractions/ref/Microsoft.Extensions.Configuration.Abstractions.cs

```diff
@@ -26,16 +26,17 @@ public static partial class ConfigurationPath
         public static readonly string KeyDelimiter;
         public static string Combine(System.Collections.Generic.IEnumerable<string> pathSegments) { throw null; }
         public static string Combine(params string[] pathSegments) { throw null; }
-        public static string GetParentPath(string path) { throw null; }
-        public static string GetSectionKey(string path) { throw null; }
+        public static string? GetParentPath(string? path) { throw null; }
```

Comment:
`path` being nullable sounds counter-intuitive to me. Is this documented? If not consider making this non-null

Reply:
Let's continue discussion in https://github.com/dotnet/runtime/pull/57401#discussion_r709917799